proposals/boost/boost-maven/boost-runtimes/runtime-openliberty/src/main/java/boost/runtimes/openliberty/boosters/LibertyMPOpenTracingBoosterConfig.java

```diff
@@ -29,9 +29,13 @@ public LibertyMPOpenTracingBoosterConfig(Map<String, String> dependencies, Boost
 
     @Override
     public String getFeature() {
-        if (getVersion().equals(MP_20_VERSION)) {
+        if (getVersion().equals(MP_OPENTRACING_VERSION_11)) {
```

Comment:
These tests read oddly.. maybe we should change the string returned to "FEATURE_OPENTRACING_11" ?